bentoml/deployment/clipper/__init__.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2019 Atalaya Tech, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import shutil
+
+import docker
+
+from bentoml.archive import load
+from bentoml.handlers import ImageHandler
+from bentoml.deployment.utils import generate_bentoml_deployment_snapshot_path, \
+    process_docker_api_line
+from bentoml.deployment.clipper.templates import DEFAULT_CLIPPER_ENTRY, DOCKERFILE_CLIPPER
+from bentoml.utils.exceptions import BentoMLException
+
+
+def build_docker_image(snapshot_path):
+    docker_api = docker.APIClient()
+    tag = "something"
+    for line in docker_api.build(path=snapshot_path, dockerfile='Dockerfile-clipper', tag=tag):
+        process_docker_api_line(line)
+    return tag
+
+
+def deploy_bentoml(clipper_conn, archive_path, api_name):
+    """Deploy bentoml bundle to clipper cluster
+
+    :param clipper_conn: Clipper connection.
+    :param archive_path: String, Path to bentoml bundle, it could be local filepath or s3 path
+    :param api_name: String, Name of the api that will be running in the clipper cluster
+    """
+    bento_service = load(archive_path)
+    apis = bento_service.get_service_apis()
+
+    if api_name:
+        api = next(item for item in apis if item.name == api_name)
+    elif len(apis) == 1:
+        api = apis[0]
+    else:
+        raise BentoMLException(
+            'Please specify api-name, when more than one API is present in the archive')
+    application_name = bento_service.name + '-' + api.name
+    input_type = 'strings'
```

Comment:
Why do we still need 'handle_clipper_numbers'? looks like its just strings and bytes we are supporting?